heatpump.yaml

```diff
@@ -1070,119 +1088,119 @@ sensor:
       return t1s_dhw;
 
 binary_sensor:
-  # Register: 0
+  # Register: 0 ->Is read as sensor from modbus to be fully switch functiuoning
   # Bit: 0 -> Is present in this config as a 'switch'
   # Bit: 1 -> Is present in this config as a 'switch'
   # Bit: 2 -> Is present in this config as a 'switch'
   # Bit: 3 -> Is present in this config as a 'switch'
-  # Bit: 4
```

Comment:
The "reserved" bits should not be removed. Although I do not know what they are, but since they are valid registers, it is good to have the history of these, so that anyone is able to see if a register is changed.

Reply:
Yes -it can exists but in my oppinion good to be commented for compilation reserved as not used at moment -maybe in future in 1% chance ;)